components/dataproducts/dataset/useColumnTags.tsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from 'react'
+import {
+  BigQueryType,
+  useGcpGetColumnsQuery,
+  DatasetQuery,
+} from '../../../lib/schema/graphql'
+
+export type PIITagType =
+  | 'PII_DirekteIdentifiserende'
+  | 'PII_SærligKategori'
+  | 'PII_KanVæreIndirekteIdentifiserende'
+
+export const PIITagNames = new Map([
+  ['PII_DirekteIdentifiserende', 'Direkte Identifiserende'],
+  ['PII_SærligKategori', 'Særlig Kategori'],
```

Comment:
```suggestion
  ['PII_DirekteIdentifiserende', 'Direkte identifiserende'],
  ['PII_SærligKategori', 'Særlig kategori'],
  ```
  
  my only real nitpick here :)